src/main/resources/templates/db_scripts/schema.json

```diff
@@ -4,7 +4,7 @@
       "tableName": "tags",
       "withMetadata": false,
       "pkColumnName": "id",
-      "generateId": true,
+      "generateId": false,
```

Comment:
Just curious as to why we changed this?

Reply:
I have found that we have test data duplication during the enabling/disabling module for the test purposes. As we leave PK creation to DB, the script will create new PKs for our entities and add them to already existing rows. Unfortunately, schema deletion is not performed on a disabling module for the tenant but performed on tenant deletion as it mentioned in doc https://github.com/folio-org/raml-module-builder#the-delete-tenant-api. This way is not convenient for us, that is why I have changed pk generation attribute to false.

Reply:
Thanks for the detailed information @natali-zaitseva